### PR TITLE
Merge the two conftest.py files

### PR DESCRIPTION
**Pull request by kinow**
This is a small change with no associated Issue. Issues found while re-basing #151 

We have two `conftest.py` files. One in the `cylc.uiserver.tests` module, another in the `cylc.uiserver` module. The older one is from `cylc.uiserver.tests`, and since the contents of `conftest.py` is used only by test, I've merged the two files, removing `cylc.uiserver.conftest`.

Tested as follows:

1. ran the tests with `pytest`, all good
2. ran again now after commenting out the contents of `cylc.uiserver.conftest`

```bash
Exception ignored in: <coroutine object Queue.get at 0x7fea8da87050>
Traceback (most recent call last):
  File "/home/kinow/Development/python/miniconda3/lib/python3.7/asyncio/queues.py", line 161, in get
    getter.cancel()  # Just in case getter is not done yet.
  File "/home/kinow/Development/python/miniconda3/lib/python3.7/asyncio/base_events.py", line 691, in call_soon
    self._check_closed()
  File "/home/kinow/Development/python/miniconda3/lib/python3.7/asyncio/base_events.py", line 479, in _check_closed
    raise RuntimeError('Event loop is closed')
RuntimeError: Event loop is closed

```

3. moved the event_loop fixture to `cylc.uiserver.tests.conftest` and deleted `cylc.uiserver.conftest`
4. ran the tests again, all good

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Already covered by existing tests.
- [x] No change log entry required (why? e.g. invisible to users).
- [x] No documentation update required.
- [x] No dependency changes.

### CODE REVIEW

**Comment by kinow** [comment]
(also fixed typos, and unused imports)

File: cylc/uiserver/tests/test_workflows_mgr.py [hunk 1]
```diff
@@ -13,23 +13,16 @@
...
-from itertools import product
-from pathlib import Path
-import pytest
 from random import random
 
-from pytest_mock import MockFixture
-
-from cylc.flow import ID_DELIM
-from cylc.flow.exceptions import ClientTimeout
-from cylc.flow.network import API
```
Comment (kinow) [inline cylc/uiserver/tests/test_workflows_mgr.py:25]: Unused imports

File: cylc/uiserver/tests/conftest.py [hunk 2]
```diff
@@ -43,3 +43,16 @@ async def async_request(self, command, args=None, timeout=None):
...
+def event_loop():
+    """This fixture defines the event loop used for each test."""
+    loop = asyncio.get_event_loop_policy().new_event_loop()
+    yield loop
+    # gracefully exit async generators
+    loop.run_until_complete(loop.shutdown_asyncgens())
+    # cancel any tasks still running in this event loop
+    for task in asyncio.all_tasks(loop):
+        task.cancel()
+    loop.close()
```
Comment (kinow) [inline cylc/uiserver/tests/conftest.py:58]: Copied as-is from the other `conftest.py` file.

**Comment by kinow** [comment]
As for test coverage, the decrease is because `cylc.uiserver.conftest.py` is included in the coverage reports (we exclude only the `tests` module).

image

So no effective code coverage reduction :+1:

**Review by hjoliver — APPROVED** [review]
LGTM :+1:

**Review by dwsutherland — APPROVED** [review]
Tested and found working, LGTM 👍

**Pull request merged**